### PR TITLE
doc(frontend) : added Frontend documentation

### PR DESCRIPTION
**Pull request by Arielpetit**

### CODE REVIEW

**Review by Motouom — CHANGES_REQUESTED** [review]
The name of the application is PackIt and not Packit. please change it everywhere in your documentation.

File: doc/Frontend-Documentation.md [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+
+Packit allows users to:
+- **Shorten Links**: Quickly convert long URLs into shorter, more manageable links.
+- **Customize Shortened Links**: Personalize the shortened links to make them more memorable and relevant.
+- **Generate QR Codes**: Create QR codes for the shortened links, facilitating easy sharing and access via mobile devices.
+- **Analyze Link Performance**: Access both detailed and general analytics about the shortened links, including metrics such as click counts, geographic data, and referral sources.
+
+Packit is designed to be user-friendly, enabling users to manage their links efficiently and gain valuable insights into their link traffic.
+
+## Functionality of the Project
```
Comment (Motouom) [inline doc/Frontend-Documentation.md:17]: Functionalities instead of functionality

File: doc/Frontend-Documentation.md [hunk 2]
```diff
@@ -0,0 +1,628 @@
...
+
+### Link Shortening
+Packit offers a straightforward process for shortening long URLs. Users simply paste their lengthy URLs into the application, and with a single click, Packit generates a shortened link. This shortened link retains the original destination but is significantly reduced in character count, making it easier to share and manage.
```
Comment (Motouom) [inline doc/Frontend-Documentation.md:20]: into the application, and with a single  when you use a "and", no need to put a comma before

File: doc/Frontend-Documentation.md [hunk 3]
```diff
@@ -0,0 +1,628 @@
...
+### Centralized Link Management
+Users can manage all their shortened links in one central location within the Packit application. This feature allows users to:
+
+- **Edit**: Modify the original or shortened URLs and update the customized text.
+- **Copy**: Quickly copy the shortened links to the clipboard for easy sharing.
+- **Delete**: Remove links that are no longer needed or relevant.
+
+Centralized link management provides users with an organized and efficient way to handle all their links, ensuring they can easily perform necessary operations.
+
+## Technologies Used
```
Comment (Motouom) [inline doc/Frontend-Documentation.md:46]: Please endeavor to precise the versions of the technologies that are being used

**Review by Motouom — CHANGES_REQUESTED** [review]

**Review by Calebasah — APPROVED** [review]

**Review by Motouom — APPROVED** [review]
Nice work, the front-end images, app logo and other diagrams that have to be included.

**Review by mbunwe-victor — APPROVED** [review]
Good job

**Pull request merged**